Documentation/teaching/labs/kernel_modules.rst

```diff
@@ -974,7 +974,7 @@ copy the module to the VM, by running the following commands in
   $ make build
   $ make copy
```

Comment:
Changes look good to me. One minor comment: Please always add a prefix to the commit message stating the kernel 'subsystem' touched by the change.

Also, please use a verb in the commit message. For this commit, we should have something like this:

```
Documentation: Fix a wording problem
```

Reply:
Understood